en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
     </tbody>
     </table>
 
-    ![](../../../assets/img/Learn/yahooweatherapi.png)
+    ![](../../../assets/img/Learn/createpizzashack.png)
```

Comment:
Shall we use the {{base_path}} when referring to images?

Reply:
+1